integrations/jenkins/.port/resources/blueprints.json

```diff
@@ -0,0 +1,94 @@
+[
+    {
+        "identifier": "jenkinsJob",
+        "description": "This blueprint represents a job event from Jenkins",
+        "title": "Jenkins Job",
+        "icon": "Jenkins",
+        "schema": {
+            "properties": {
+                "jobName": {
+                    "type": "string",
+                    "title": "Project Name"
+                },
+                "jobStatus": {
+                    "type": "string",
+                    "title": "Job Status",
+                    "enum": ["created", "updated", "deleted"],
+                    "enumColors": {
+                        "created": "green",
+                        "updated": "yellow",
+                        "deleted": "red"
+                    }
+                },
+                "timestamp": {
+                    "type": "string",
+                    "format": "date-time",
+                    "title": "Timestamp",
+                    "description": "Last updated timestamp of the job"
+                },
+                "url": {
+                    "type": "string",
+                    "title": "Project URL"
+                }
+            },
+            "required": []
+        },
+        "mirrorProperties": {},
+        "calculationProperties": {},
+        "relations": {
+            "jobs": {
+                "title": "Pipelines",
+                "target": "jenkinsJob",
+                "required": false,
+                "many": true
+            }
+        }
+    },
+    {
+        "identifier": "jenkinsBuild",
+        "description": "This blueprint represents a build event from Jenkins",
+        "title": "Jenkins Build",
+        "icon": "Jenkins",
+        "schema": {
+            "properties": {
+                "buildStatus": {
+                    "type": "string",
+                    "title": "Build Status",
+                    "enum": ["SUCCESS", "FAILURE", "UNSTABLE"],
+                    "enumColors": {
+                        "SUCCESS": "green",
+                        "FAILURE": "red",
+                        "UNSTABLE": "yellow"
+                    }
+                },
+                "buildUrl": {
+                    "type": "string",
+                    "title": "Build URL",
+                    "description": "URL to the build"
+                },
+                "timestamp": {
+                    "type": "string",
+                    "format": "date-time",
+                    "title": "Timestamp",
+                    "description": "Last updated timestamp of the build"
+                },
+                "buildDuration": {
+                    "type": "number",
+                    "title": "Build Duration",
+                    "description": "Duration of the build"
+                }
+            },
+            "required": []
+        },
+        "mirrorProperties": {},
+        "calculationProperties": {},
+        "relations": {
+            "job": {
```

Comment:
use better title and identifiers to explain what this is

Reply:
changed this to `parentJob`